site/src/scaffold/scripts/popover.ts

```diff
@@ -0,0 +1,37 @@
+export function initPopover(container: HTMLElement) {
+	container.addEventListener('click', event => {
+		const popovers: HTMLDetailsElement[] = Array.from(
+			container.querySelectorAll('details.popover')
+		);
+
+		if (popovers.length === 0) {
+			return;
+		}
+
+		popovers.forEach(p =>
```

Comment:
Does this add new event listeners on each subsequent click?

Reply:
also consider, for ... of loop instead

Reply:
I don't think the loop will be required. This code from dropdown.ts is still good and covers touch, virtual dom, and edge cases with iframes.
https://dev.azure.com/ceapex/Engineering/_git/docs-ui?path=/packages/scripts/src/dropdown.ts&version=GBdevelop&line=38&lineEnd=90&lineStartColumn=1&lineEndColumn=49&lineStyle=plain&_a=contents

Reply:
Changed the approach, please revisit. I don't assign the key event listener to every popover with random clicks on a page.

Reply:
Revisit the link Jeremy posted above?